src/migration_utilities.js

```diff
@@ -0,0 +1,18 @@
+const argv = require("yargs")
+  .option("onlyMigrateStableX", {
+    describe: "Allows to restrict the migration only to StableX"
+  })
+  .option("onlyMigrateSnappAuction", {
+    describe: "Allows to restrict the migration only to SnappAuction"
+  })
+  .help(false)
+  .version(false)
+  .argv
+
+function isMigrationRequired(network) {
```

Comment:
maybe this method could be more general `isDevelopmentNetwork` and then reused in the other PR when we try to get the artifact (I saw a similar if statement there).